packages/airnode-examples/integrations/authenticated-coinmarketcap/create-secrets.ts

```diff
@@ -0,0 +1,25 @@
+import prompts, { PromptObject } from 'prompts';
+import { getCommonSecrets, writeSecrets } from '../utils';
+import { runAndHandleErrors } from '../../src';
+
+const questions: PromptObject[] = [
+  {
+    type: 'text',
+    name: 'apiKey',
+    message: [
+      'The CoinMarketCap example requires an API key, which can be acquired with a free, Basic account.',
+      'Please enter your CoinMarketCap API key',
```

Comment:
```suggestion
      'Please enter your CoinMarketCap API key (it will never leave this machine)',
```

Reply:
I am not sure whetehr we should do this or not. WDYT?

Reply:
>Actually, this is not enough - also the users need to run this to compile the contracts.
We should add this as a step to the "main" README

Isn't this covered by Setup? - https://github.com/api3dao/airnode/blame/master/packages/airnode-examples/README.md#L26-L29

> I am not sure whetehr we should do this or not. WDYT?

Is it true the API key doesn't leave the machine even if the user chooses the `aws` flow? I admit I don't know the inner workings of how secrets are handled by the deployer.

Reply:
> Is it true the API key doesn't leave the machine even if the user chooses the aws flow? I admit I don't know the inner workings of how secrets are handled by the deployer.

You're right - they do